testnets/testnets.md

```diff
@@ -53,6 +73,28 @@ wasmcli query wasm list-code
 wasmcli keys add mywallet
 ```
 
+# Joining to be launched testnets
+
+You need to have your address and informations defined in networks genesis file to join for not yet launched testnets.
+Here is the script you can run to handle it automatically. It uses `coral` [network specific executables](https://github.com/CosmWasm/testnets/tree/master/coral#coral-wip):
+
+```sh
+cd $CW_DIR
+git clone git@github.com:CosmWasm/testnets
```

Comment:
Please note they will have to fork this to their own github account as they cannot push branches to our git